foundation/foundation-mda/src/main/resources/templates/data-delivery-common/pipeline.chart.pom.configuration.vm

```diff
@@ -13,9 +13,7 @@
                 <executable>helm</executable>
                 <arguments>
                     <argument>template</argument>
```

Comment:
I: you cannot use a `repo` with OCI b/c there isn't an index or way to search what is in an OCI repository

Reply:
This can be handled in a later ticket. But have we thought about how downstream project will be incorporating these changes when they upgrade?

Reply:
Good question - Yes. This change will handle baseline charts.  They can use our public charts or re-wire this to their own mirror/proxy (e.g., Nexus, Artifactory).  Another ticket will allow specification of a totally different repo as the project-default chart repository.